VSRAD.Package/BuildTools/Errors/Parser.cs

```diff
@@ -104,7 +104,7 @@ private static MessageKind ParseMessageKind(string kind)
         {
             switch (kind)
             {
-                case "E": case "error": case "ERROR": return MessageKind.Error;
+                case "E": case "error": case "ERROR": case "fatal error": return MessageKind.Error;
```

Comment:
Clang error messages always display kind in lowercase, so this is not a case. We won't get here with `Error` or `Warning` because of [ClangErrorRegex](https://github.com/vsrad/radeon-asm-tools/blob/ab7ab5962bc0a8252f44a88acdd7401a1289e88d/VSRAD.Package/BuildTools/Errors/Parser.cs#L40)